services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -788,7 +798,7 @@ private void doDumpPortDescriptionRequest(
                     .findFirst()
                     .orElseThrow(() -> new SwitchOperationException(
                             DatapathId.of(switchId.toLong()),
-                            String.format("Port %d does not exist on the switch %s", port, switchId)));
+                            format("Port %d does not exist on the switch %s", port, switchId)));
```

Comment:
same here...
We should always represent port as switchId() + "-" + portNumber().